persistence/scaladsl/src/test/scala/com/lightbend/lagom/scaladsl/persistence/TestEntity.scala

```diff
@@ -43,9 +43,9 @@ object TestEntity {
     override def aggregateTag: AggregateEventShards[Evt] = Evt.aggregateEventShards
   }
 
-  final case class Appended(element: String) extends Evt
+  final case class Appended(entityId: String, element: String) extends Evt
```

Comment:
It's not strictly needed, however PersistentEntityRegistry.eventStream returns a tuple of the event and offset.  Events are not typically useful without their entity ID.

It was also needed initially when I implemented the sharding, as it was the events responsibility to select the shard from the the entity id via the aggregateTag method, and in fact that's why I put the entity ID in here originally. Now it can return a sharded tag, and PersistentEntityActor will calculate the shard and it knows the entity ID so it can do that.

But for the first reason, it will still practically always be necessary to include the entity ID.  If we wanted to change that, we'd have to make `PersistentEntityRegistry.eventStream` return an object the captured the entity ID, offset, and event itself, rather than a tuple of the offset and event.  Maybe since the Scala API is new, we could do that straight off, for the Java API we may need to create a new method.